### PR TITLE
fix: advanced modal on appointment search

### PR DESCRIPTION
**Pull request by FrancescoGottardi**

### CODE REVIEW

File: src/view/search/parts/advanced-filter-button.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React, { FC } from 'react';
+import { Button, Container, Tooltip } from '@zextras/carbonio-design-system';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+
+const BorderContainer = styled(Container)`
+	border-bottom: 1px solid ${({ theme }): string => theme?.palette?.gray2?.regular};
+	border-right: 1px solid ${({ theme }): string => theme?.palette?.gray2?.regular};
```
Comment (dhavaldodiya) [inline src/view/search/parts/advanced-filter-button.tsx]: Please use rem instead of px

Comment (FrancescoGottardi) [reply src/view/search/parts/advanced-filter-button.tsx]: changed, tnx!

**Review by dhavaldodiya — DISMISSED** [review]

File: src/view/search/advance-filter-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+					!includes(queryArray, v.label) &&
+					!/^Is:/.test(v.label) &&
+					!v.isQueryFilter
+			),
+			(q) => ({ ...q, hasAvatar: false })
+		);
+		setOtherKeywords(updatedQuery);
+	}, [query, queryArray]);
+    
+    const DEFAULT_DATE_START = moment().startOf('day').subtract(1, 'months').valueOf();
```
Comment (nubsthead) [inline src/view/search/advance-filter-modal.tsx]: move these two constants in the constants directory in a separated file so you can use it everywhere.

File: src/view/search/search-view.tsx [hunk 3]
```diff
@@ -82,13 +84,15 @@ const SearchView: FC<SearchProps> = ({ useQuery, ResultsHeader }) => {
 		[searchInFolders]
 	);
 
+	const [spanStart, setSpanStart] = useState(() =>
+		moment().startOf('day').subtract(1, 'months').valueOf()
```
Comment (nubsthead) [inline src/view/search/search-view.tsx]: Use the constants also here

**Review by nubsthead — CHANGES_REQUESTED** [review]

File: src/view/search/advance-filter-modal.tsx (hunk 2 above)
Comment (FrancescoGottardi) [reply src/view/search/advance-filter-modal.tsx]: moved, tnx!

**Review by giuliano176 — APPROVED** [review]

**Review by dhavaldodiya — APPROVED** [review]

**Review by gnekoz — APPROVED** [review]

**Review by nubsthead — APPROVED** [review]

**Pull request merged**